packages/frontend/src/hooks/useRealityQuestionResponses.ts

```diff
@@ -51,6 +52,7 @@ export function useRealityQuestionResponses(
         provider,
         questionId,
         ipfsGatewayURL,
+        blockNumber.data,
```

Comment:
Why is this required? The idea is that if we're calling this hook the question has been finalized, so there's no need to watch for new responses. This is a one and done hook.

Reply:
It's needed to update the `finalizing in` timestamp since we need to get the latest response each time.

```ts
const mostRecentResponseTimestamp = responses.length > 0 ? responses[responses.length - 1].timestamp : 0;
```